deploy/ansible/roles/elasticsearch/files/elasticsearch_settings.yml

```diff
@@ -2,19 +2,41 @@
 settings:
   analysis:
     analyzer:
-      nyaapantsu_analyzer:
-        tokenizer: nyaapantsu_tokenizer
+      nyaapantsu_search_analyzer:
+        type: custom
+        tokenizer: standard
+        char_filter:
+          - my_char_filter
         filter:
           - standard
           - lowercase
-    tokenizer:
-      nyaapantsu_tokenizer:
-        type: edge_ngram
+      nyaapantsu_index_analyzer:
+        type: custom
+        tokenizer: standard
+        char_filter:
+          - my_char_filter
+        filter:
+          - resolution
+          - lowercase
+          - my_ngram
+          - word_delimit
+    filter:
+      my_ngram:
+        type: edgeNGram
         min_gram: 2
         max_gram: 15
-        token_chars:
-          - letter
-          - digit 
+      resolution:
+        type: pattern_capture
+        patterns: ["(\\d+)[xX](\\d+)"]
+      word_delimit:
+        type: word_delimiter
+        preserve_original: true
+        split_on_numerics: false
+    char_filter:
+      my_char_filter:
+        type: mapping
+        mappings: ["-=>_", "!=>_", "_=>\\u0020"]
```

Comment:
Since this is the ES config used by nyaa.si, doesn't that mean you'll have to switch the license to GPL?